doc/sphinx-guides/source/locale/pl/LC_MESSAGES/user/dataverse-management.po

```diff
@@ -290,172 +393,205 @@ msgid ""
-msgstr ""
+msgstr "Role i uprawnienia"
 
-#: ../../source/user/dataverse-management.rst:97
+#: ../../source/user/dataverse-management.rst:90
 msgid ""
 "Dataverse user accounts can be granted roles that define which actions "
 "they are allowed to take on specific dataverses, datasets, and/or files. "
 "Each role comes with a set of permissions, which define the specific "
 "actions that users may take."
 msgstr ""
+"W repozytorium kontom użytkowników można przypisywać role. Rola określa, j"
+"akie działania może  podejmować użytkownik w odniesieniu do wskazanych kol"
+"ekcji, zbiorów danych i/lub plików. Z każdą rolą związany jest zestaw upra"
+"wnień określających jakie działania mogą podejmować użytkownicy."
 
-#: ../../source/user/dataverse-management.rst:99
+#: ../../source/user/dataverse-management.rst:92
 msgid ""
 "Roles and permissions may also be granted to groups. Groups can be "
 "defined as a collection of Dataverse user accounts, a collection of IP "
 "addresses (e.g. all users of a library's computers), or a collection of "
 "all users who log in using a particular institutional login (e.g. "
 "everyone who logs in with a particular university's account credentials)."
 msgstr ""
+"Role i uprawnienia mogą być przyznawane również grupom. Grupy można zdefin"
+"iować jako zbiór kont użytkowników repozytorium, zbiór adresów IP (np. wsz"
+"yscy użytkownicy komputerów w sieci biblioteki) lub zbiór wszystkich użytk"
+"owników, którzy logują się z konkretnej instytucji za pomocą logowania fed"
+"eracyjnego."
 
-#: ../../source/user/dataverse-management.rst:101
+#: ../../source/user/dataverse-management.rst:94
 msgid ""
 "Admins of a dataverse can assign roles and permissions to the users of "
 "that dataverse. If you are an admin on a dataverse, then you will find "
 "the link to the Permissions page under the Edit dropdown on the dataverse"
 " page."
 msgstr ""
+"Role i związane z nimi uprawnienia mogą przyznawać administratorzy kolekcj"
+"i. Jeśli jesteś administratorem kolekcji, link do strony \"Uprawnienia\" z"
+"najdziesz na stronie kolekcji, w menu rozwijanym pod przyciskiem \"Edytuj\"."
 
-#: ../../source/user/dataverse-management.rst:107
+#: ../../source/user/dataverse-management.rst:100
 msgid "Permissions link highlighted"
-msgstr ""
+msgstr "Pośdwietlony link do uprawnień"
 
-#: ../../source/user/dataverse-management.rst:107
+#: ../../source/user/dataverse-management.rst:100
 msgid "Screenshot of dataverse page with highlighted Permissions link location."
-msgstr ""
+msgstr "Zrzut ekranu podstrony repozytorium z podświetlonym linkiem do uprawnień."
 
-#: ../../source/user/dataverse-management.rst:109
+#: ../../source/user/dataverse-management.rst:102
 msgid ""
 "Clicking on Permissions will bring you to the page presented in "
 ":numref:`dvperms2`:"
 msgstr ""
+"Kliknięcie na \"Uprawnienia\" spowoduje przeniesienie na tę stronę:"
+#This will generate warning in build time about :numref: being translated to nothing.
```

Comment:
We could update to a newer version of shpinx (4.5 or later) which would allow to suppress the warning ( https://github.com/sphinx-doc/sphinx/blob/a001bf47d66ae804a9a6e5d754de9b5eda4d0eb9/CHANGES#L30 )